<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QWidget" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyword Replacement in File Path</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="Label9">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%M</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="Label20">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Minute (00-59)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%m</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Label4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%d</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Label5">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%n</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="Label19">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Hour (00-23)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="Label16">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Day of Year (ddd)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="Label14">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Month (mm)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="Label6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%W</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="Label17">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: GPS Week No (wwww)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="Label12">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Year (yyyy)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="Label26">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: 6H Hour (00,06,12,18)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="Label30">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Station ID</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="Label29">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%r</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="Label36">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Station ID (UPPER case)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="Label35">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%S</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="Label24">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: 3H Hour (00,03,...,21)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="Label22">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Hour Code (a,b,...,x)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="Label34">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Station ID (lower case)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="Label13">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Year (yy)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="Label31">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%b</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="Label32">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Base Station ID</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="Label25">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%hb</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="Label11">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%H</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="Label23">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%ha</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="Label8">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%h</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Label2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%y</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Label1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%Y</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="Label27">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%hc</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label7">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%D</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="Label18">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Day of Week (0-6)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="Label33">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="Label28">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: 12H Hour (00,12)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="Label15">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Day of Month (dd)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="Label10">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%S</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="Label21">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Second (00:59)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="Label37">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>%N</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="Label38">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>: Sequence Number</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
